ui/shared/components/panel/search/LocusListItemsFilter.jsx

```diff
@@ -14,28 +14,34 @@ const PA_ICON_PROPS = Object.entries(PANEL_APP_CONFIDENCE_LEVELS).reduce((acc, [
 
 const SUBSCRIPTION = { values: true }
 
-const LocusListItemsFilter = ({ shouldShow, iconColor, label, ...props }) => (
+const LocusListItemsFilter = ({ shouldShow, shouldDisable, iconColor, label, filterComponent, ...props }) => (
   <FormSpy subscription={SUBSCRIPTION}>
-    {({ values }) => shouldShow(values?.search?.locus || {}) && (
-      <BaseSemanticInput
-        inputType="TextArea"
-        rows={8}
-        label={PA_ICON_PROPS[iconColor] ? (
-          <label>
-            <ColoredIcon {...PA_ICON_PROPS[iconColor]} />
-            {label}
-          </label>
-        ) : label}
-        {...props}
-      />
+    {({ values }) => (!shouldShow || shouldShow(values?.search?.locus || {})) && createElement(
+      filterComponent || BaseSemanticInput, {
+        inputType: 'TextArea',
+        inline: true,
+        rows: 8,
+        label: (
+          PA_ICON_PROPS[iconColor] ? (
+            <label>
+              <ColoredIcon {...PA_ICON_PROPS[iconColor]} />
+              {label}
+            </label>
+          ) : label
+        ),
+        disabled: shouldDisable(values?.search?.locus || {}),
+        ...props,
+      },
     )}
   </FormSpy>
 )
 
 LocusListItemsFilter.propTypes = {
-  label: PropTypes.string,
+  label: PropTypes.node,
```

Comment:
Does the `label` need to be a `node`? What I can find in this PR is that all the `labels` are strings.

Reply:
`node` is a prop type that includes any valid react child node, including plain strings and jsx. Before, all the labels were plain strings so this prop type made sense, but the label for variant search box which is now using this component has jsx in it so we need to change the prop type to support that